src/sparseml/exporters/transforms/kv_cache/configs.py

```diff
@@ -113,8 +113,8 @@ class Config:
     additional_transforms=AdditionalTransformsMPT,
     key_num_attention_heads="n_heads",
     key_num_embedding_hidden_size="d_model",
-    transpose_value_input=None,
-    transpose_key_input=(0, 1, 3, 2),
+    transpose_value_input=(0, 2, 1, 3),
+    transpose_key_input=(0, 2, 1, 3),
```

Comment:
Have we verified that this will work properly for any previously injected KV Cache models?
Could you add to the PR description, which models you tested this change with?